manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```diff
@@ -21,6 +22,32 @@ images:
 patchesStrategicMerge:
   - patches/remove-namespace.yaml
 
+patches:
+  # Extend RBAC permission list of katib-ui so it can
+  # create SubjectAccessReview resources.
+  - target:
+      kind: ClusterRole
+      name: katib-ui
+      group: rbac.authorization.k8s.io
+      version: v1
+    path: patches/ui-rbac.yaml
+  # Enable RBAC authz checks in UI's backend.
+  - target:
+      version: v1
+      kind: Deployment
+      name: katib-ui
+    patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: APP_DISABLE_AUTH
+          value: "false"
```

Comment:
Should we move this patch to a separate file, the same as other patches?

https://github.com/kubeflow/katib/tree/master/manifests/v1beta1/installs/katib-leader-election/patches

Reply:
Fixed!